NoteFind/View Controllers/NewNoteViewController.m

```diff
@@ -37,8 +33,8 @@ - (void)setImportMenu:(UIMenu *)importMenu {
     NSMutableArray *options = [[NSMutableArray alloc] init];
     [options addObject:[UIAction actionWithTitle:@"import from files" image:nil identifier:nil handler:^(__kindof UIAction * _Nonnull action) {
         
-        UTType *const UTTypePDF;
-        UIDocumentPickerViewController *documentPicker = [[UIDocumentPickerViewController alloc] initForOpeningContentTypes:@[UTTypePDF]];
+        NSArray<UTType *> *types = @[UTTypeText];
+        UIDocumentPickerViewController *documentPicker = [[UIDocumentPickerViewController alloc] initForOpeningContentTypes:types];
```

Comment:
Given you have a fixed single element array here, you can probably just inline it into the next line
```suggestion
        UIDocumentPickerViewController *documentPicker = [[UIDocumentPickerViewController alloc] initForOpeningContentTypes:@[UTTypeText]];
```